velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -479,7 +479,12 @@ TEST_F(HiveDataSinkTest, hiveBucketProperty) {
 
 TEST_F(HiveDataSinkTest, basic) {
   const auto outputDirectory = TempDirectoryPath::create();
-  auto dataSink = createDataSink(rowType_, outputDirectory->getPath());
+  auto fileFormat = dwio::common::FileFormat::DWRF;
```

Comment:
In these tests here we do want to keep DWRF and add PARQUET.

We might wrap the main test body into a loop and add a trace to say which fileFormat we are using.

There might be more elegant ways of doing this.

Reply:
Will implement your suggestion, thanks!